lib/vanagon/platform/solaris_10.rb

```diff
@@ -42,14 +42,16 @@ def generate_package(project)
           # - The bin directory and all bin files are owned by root:bin instead of root:sys
           # - All files under lib are owned by root:bin instead of root:sys
           # - All .so files are owned by root:bin instead of root:sys
+          # - Explicity only include puppet directories 
+          #   (this should exclude things like root/bin root/var and such)
           %((cd $(tempdir)/#{name_and_version}; pkgproto . | sort | awk ' \
-            $$1 ~ /^d$$/ {print "d",$$2,$$3,"0755 root sys";} \
+            $$1 ~ /^d$$/ && $$3 ~ /^.*puppet.*/ {print "d",$$2,$$3,"0755 root sys";} \
```

Comment:
Yeah, I just found that function actually, thanks @haus, I was thinking this might be bad and I hoped someone would point me in the right direction when I opened the PR.